sdk/src/main/AndroidManifest.xml

```diff
@@ -26,7 +26,6 @@
             android:permission="android.permission.BIND_JOB_SERVICE"
             android:exported="true" />
         <service android:name=".RadarForegroundService"
-            android:foregroundServiceType="location"
-            android:stopWithTask="true" />
+            android:foregroundServiceType="location" />
```

Comment:
I don't know why it was done this way. This flag was set when this service was first added to the SDK. @nickpatrick do you know the rationale for adding this flag?

Reply:
This flag ties the SDK's Foreground service with any Activity or UI of the app that uses the SDK. So for Any app that uses CONTNUOUS mode, when someone _swipes away_ their Activity (ie kills the app), Radar's CONTINUOUS tracking will immediately stop working. Since the SDK is not meant to be tied to any specific UI, it does not make sense to kill the service when the activity is killed.

Reply:
hmm, I was seeing the fg service persist even after I force killed toolkit, here's the Slack thread comparing my observations of iOS's blue bar with Android's fg service in continuous mode - https://radarlabs.slack.com/archives/C02MF6PAWBH/p1644266051162629

Reply:
@jsani-radar I was seeing different behavior testing locally. Wonder if this is something with different behaviors based on Android version. Other thing to consider is that the fg service comes back online after the next location update. My observations on Android 12 before these changes was that swiping away toolkit caused the fg service to disappear, but after making this change it worked fine. Differences in behaviors on different API levels would align with this different behaviors, and potentially would make sense with why it is hard to trace on capacitor as well. @brettguenther what Android version is having errors on capacitor?

Reply:
ah interesting, for ref my Android device is on 9 or 10